app/src/main/java/com/kickstarter/ui/viewholders/RootCommentViewHolder.kt

```diff
@@ -25,9 +25,16 @@ class RootCommentViewHolder(
                 CommentCardStatus.values().firstOrNull { commentCardData.commentCardState == it.commentCardStatus }?.let {
                     if (it == CommentCardStatus.CANCELED_PLEDGE_MESSAGE) {
                         binding.commentsCardView.setCommentCardStatus(it)
-                        binding.commentsCardView.setCancelPledgeMessage(
-                            context().getString(R.string.This_person_canceled_their_pledge).plus(" ").plus(context().getString(R.string.Show_comment))
-                        )
+
+                        if (vm.isCommentModerationEnabled == true) {
```

Comment:
instead of having a variable on the viewModel, you could just consult `this.environment().optimizely()...` locally on the viewHolder, you have it available